test/controllers/v1/elections_controller_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class V1::ElectionsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @election = elections(:one)
+  end
+
+  test "should find election by id" do
+    get v1_election_url(@election),
+        headers: {
+          "Accept": "application/vnd.api+json"
+        }
+
+    assert_equal "application/vnd.api+json", @response.content_type
```

Comment:
Not sure if this test is doing what it says? is this not missing an `assert_response :ok`